client/my-sites/domains/domain-management/components/breadcrumbs/style.scss

```diff
@@ -171,6 +171,10 @@
 .breadcrumbs__buttons-mobile {
 	display: flex;
 
+	> * + * {
```

Comment:
Nice one!
There's a relatively new flex/grid display CSS property called [`gap`](https://developer.mozilla.org/en-US/docs/Web/CSS/gap#flex_layout) that does the same thing - and you can also define `row-gap` and `column-gap` separately on it. 

I saw a few references to it in our codebase, although I'm not so sure how that would be interpreted by our webpack loaders. It might be worth giving a try 😸

Reply:
I think it was Del who wrote that selector, it's neat 😄

I was going to say that selector isn't inside a grid context, but I just learned that `gap` can be used in flexbox too, that's nice!

Reply:
Yeah...I think you should be able to use `gap` here.  Probably more readable than the "owl-face" selector I used. :-D

Reply:
🦉 Owl face, that's a nice one `* + *` 😄